### PR TITLE
First pass on qa process.

### PR DESCRIPTION
**Pull request by patrickberkeley**

### CODE REVIEW

File: qa-ing-a-feature.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# QA
+
+Make sure the developer has left QA instructions on the ticket. If they haven't, request them.
+
+Two passes of QA need to be done on each feature:
+
+1. Test the changes following the instructions left by the developer -- the happy path -- to be certain the feature works as the developer intended.
+1. Test the changes *not* by following the developer's instructions, but rather as a 'drunk general'. This means try to break the feature and test anything and everything with the goal being to discover bugs.
```
Comment (beerlington) [inline qa-ing-a-feature.md]: What is a "drunk general"? Is that an army reference?

**Comment by alanpeabody** [comment]
LGTM, but maybe remove drunk general in favor of something generic. I totally forget the reference/story already.

**Comment by patrickberkeley** [comment]
'drunk general' reference has been removed.

**Comment by alanpeabody** [comment]
:+1:

**Pull request merged**